doc/developing.md

```diff
@@ -2,52 +2,42 @@
 
 ## npm scripts
 
-This project uses npm to run build scripts and other tasks. Here are the essential scripts;
+This project uses `npm` as an interface to run build scripts and other tasks. The scripts themselves are a combination of JavaScript, Bash and Python scripts. Here are the essential scripts;
+
+`npm install`: Installs JavaScript and Python dependencies.
+
+`npm run build`: Runs all build scripts and generates built versions of the app. You can also build specific assets with `npm run build:css`, `npm run build:js`, `npm run build:tests` and `npm run build:config` (compiles configuration files).
 
 `npm start`: Starts the app for local development. More specifically, starts a web server that serves the contents of the `/web` directory.
 
-`npm test`: Runs all tests and linting to verify code quality.
+`npm watch`: Runs all of the build tasks necessary for local development in watch mode so that the build is automatically when source files change. You should run `npm run build` first to fetch GIBS configuration. This does not generate a final build, and is only for local development.
+
+`npm test`: Runs all tests and linting to verify code quality. Make sure to run `npm run build` or `npm run build:tests` first to build a testable version of Worldview.
 
 `npm run lint`: Lints code against the project's style guides. Use `npm run lint:css` or `npm run lint:js` to lint CSS or JS files separately.
 
 `npm run e2e`: Runs end-to-end tests locally. Use `npm run e2e:chrome` or `npm run e2e:firefox` to run tests in a specific browser.
 
-`npm run browserstack`: Runs end-to-end test on Browserstack.
+`npm run browserstack`: Runs end-to-end tests on Browserstack.
 
-`npm build`: Creates JS and CSS bundles from the source and places them in `/web/build`. Use `npm run build:js` or `npm run build:css` to build JS or CSS individually.
+`npm run getcapabilites`: Makes a request to the GIBS `GetCapabilities` API and updates layer configurations.
```

Comment:
We may want to link to the GIBS API where it describes the GetCapabilities [here](https://wiki.earthdata.nasa.gov/display/GIBS/GIBS+API+for+Developers#GIBSAPIforDevelopers-OGCWebMapTileService(WMTS))

Reply:
Good idea!